### PR TITLE
Add landing page

### LINKED ISSUE #59: Design of the landing page
Currently the landing page is still empty. We need to think about what we want to show here.

https://seismo-learn.org/software/

### PR DESCRIPTION
**Pull request by core-man**
Closes #59.

### CODE REVIEW

File: source/index.rst [hunk 1]
```diff
@@ -1,6 +1,39 @@
 地震“学”软件
 ############
 
+本文档是由\ `地震“学” <https://seismo-learn.org/>`__\ 维护整理的地震学常用软件教程。
+教程主要介绍一些地震学常用软件的用途、安装、用法等，并提供一些实用示例。
+其既可以作为初学者的入门读物，也可以作为日常使用的参考资料。
```
Comment (seisman) [inline source/index.rst:6]:
```suggestion
```

File: source/index.rst [hunk 2]
```diff
@@ -1,6 +1,39 @@
...
+
+教程源码在 `GitHub <https://github.com/seismo-learn/software>`__ 上开源，
+欢迎使用这些软件的用户参与到教程的维护与更新中。
+
+软件列表
+--------
+
+.. rubric:: 地震波：
```
Comment (seisman) [inline source/index.rst:14]:
```suggestion
.. rubric:: 地震波：
```
Is "地震波" a good category?

Comment (core-man) [reply source/index.rst:13]: I mean all software related to wave propogation forward calculation.

Comment (seisman) [reply source/index.rst:13]: We can find better categories later when we have more software.

**Review by seisman — APPROVED** [review]

File: source/index.rst (hunk 2 above)
Comment (core-man) [reply source/index.rst:13]: okay~

**Pull request merged**